src/components/Layout/index.tsx

```diff
@@ -217,15 +214,40 @@ const Layout: React.FC<LayoutProps> = ({
     ]
   );
 
-  const updateConsent = (consentGranted: boolean) => {
-    const consent = consentGranted ? "granted" : "denied";
-    if (typeof window !== "undefined") {
-      window.gtag("consent", "update", {
-        ad_storage: consent,
-        analytics_storage: consent,
-      });
+  // Only show banner and Matomo if env vars are set
+  const matomoEnabled = !!GATSBY_MTM_SITE_ID && !!GATSBY_MTM_DOMAIN;
+
+  function loadMatomoScript() {
+    const u = "//www.sis.gov.uk/matomo/";
```

Comment:
Should this URL be hardcoded or held in a config or .env file somewhere? Also I feel like these variables names should be more descriptive. Would help with understanding this function